LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Khusniddin Ismoilov
```

Comment:
pleas write this instead 
Copyright (c) 2022 Osama Ashraf, Luis Muñoz,  Hevar Hoshang